InMobiAdapter/src/main/java/com/chartboost/mediation/inmobiadapter/InMobiAdapter.kt

```diff
@@ -430,13 +435,15 @@ class InMobiAdapter : PartnerAdapter {
     /**
      * Build a [BannerAdEventListener] listener and return it.
      *
+     * @param inMobiBanner An [InMobiBanner] instance that is passed down for Chartboost Mediation events.
```

Comment:
what if you did the same thing with banners as with interstitials?

Reply:
InMobi interstitials need a listener on their object creation, unlike banners. And it can't initially be null. 😞

Reply:
Interestingly, they do have a setListener method, but do we want to do that? Pass a dummy/empty one on creation, and overwrite it with the one we build? 🤔

Reply:
oh right you can't pass the thing you're making to it :P
fair enough

Reply:
Did try using an empty listener at object creation and overwrite after, but no luck after some tries. 😢 Def was worth a try, though. 😄